ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -890,8 +901,16 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </div>
             <Dropdown style={{width: '14rem'}}
                       value={this.state.workspace.billingAccountName}
-                      options={this.state.billingAccounts.map(a => ({label: a.displayName, value: a.name}))}
-                      onChange={e => this.setState(fp.set(['workspace', 'billingAccountName'], e.value))}
+                      options={this.buildBillingAccountOptions()}
+                      onChange={e => {
+                        if (e.value === CREATE_BILLING_ACCOUNT_OPTION_VALUE) {
+                          this.setState({
+                            showCreateBillingAccountModal: true
```

Comment:
This seems like kind of a dead end user flow - is there supposed to be some guidance here around refreshing the page once you've finished these steps? Or perhaps should it reload the billing account list upon closing the modal?

Reply:
I chatted with Lou and Karthik. I think the plan is to add an instruction to the zendesk page, telling the user to refresh.